sherpa/plot/backends.py

```diff
@@ -898,6 +905,12 @@ def hline(self, y, *,
         .. warning::
            No output will be produced by this backend, since the implementation
```

Comment:
Above you have changed some `warning` sections to say `function` rather than `backend`; should this one also be changed?

Reply:
The reason I changed them below is that functions are inherited by BokehBackend, so a text that says "No output will be produced by this backend" is technically wrong - the bokeh backend produces output, just not with that function. In this particular case (hline and vline) all backends we have implement their own version, so I did not have to do that. Still, it might be useful to revisit the wording and come up with something that works for both cases. It's one of those things that you only see once you write a second backend.

Reply:
I don't think it's something important (i.e. does not need to be resolved for this PR). It's just the sort of thing you notice when scanning similar changes.